src/chains/supported.chains.ts

```diff
@@ -866,6 +866,55 @@ export const supportedEVMChains: EVMChain[] = [
     },
   },
 
+  // 324 - zksync ERA
+  {
+    key: ChainKey.ERA,
+    chainType: ChainType.EVM,
+    name: 'zkSync Era',
+    coin: CoinKey.ETH,
+    id: 324,
+    mainnet: true,
+    logoURI:
+      'https://raw.githubusercontent.com/lifinance/types/main/src/assets/icons/chains/zksync.png',
+
+    metamask: {
+      chainId: prefixChainId(324),
+      blockExplorerUrls: ['https://zkscan.io/'],
+      chainName: 'ZKSYNC Erа',
```

Comment:
Why do you use another name than on https://chainlist.org/chain/324? `zkSync Era Mainnet`

Reply:
Used the ones from the researched configuration in notion, assumed they were checked alrady. But can change to the chainlist one!